src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -419,11 +419,9 @@ def _external_trade_entries(child, accumulated_trades):
             earned = round_floats_for_ui(area_data["earnedFromExternal"][k])
             results["bars"].append({
                 "energy": outgoing_energy,
-                "targetArea": child.name,
-                "energyLabel": f"{child.name} sold {abs(outgoing_energy)} kWh "
-                               f"of {k} to external consumers",
-                "priceLabel": f"{child.name} earned {earned} cents  "
-                              f"of {k} from external consumers."
+                "targetArea": k,
+                "energyLabel": f"External sources sold {abs(outgoing_energy)} kWh ",
```

Comment:
Same here, external sources are not selling anything, the root area is selling to external sources.